src/main/java/uk/gov/hmcts/reform/sendletter/tasks/MarkLettersPostedTask.java

```diff
@@ -47,6 +51,7 @@ public void run(LocalTime now) {
                 .stream()
                 .map(parser::parse)
                 .forEach(parsedReport -> {
+                    insights.trackPrintReport(parsedReport);
```

Comment:
it would be safer to log this after everything is done (few lines below) I think

Reply:
It depends what this event is tracking; is it tracking that we successfully processed a report or that we started processing a report?